GitUI/LeftPanel/RepoObjectsTree.cs

```diff
@@ -140,7 +141,7 @@ Image Pad(Image image)
 
             SearchControl<string> CreateSearchBox()
             {
-                SearchControl<string> search = new(SearchForBranch, i => { })
+                SearchControl<string> search = new(SearchForBranch, _ => { })
```

Comment:
```suggestion
                SearchControl<string> search = new(SearchForBranch, onSizeChanged: size => { })
```

Reply:
hmm.. you don't like the discard operator? We are not using the variable here. Admittedly this warning/suggestion came from Rider but it seemed reasonable.

Reply:
My main point was to add "onSizeChanged:" in order to give clarity what is ignored here. As I have looked up the declaration, I have also added "size". The discard operator would be OK to me, too.